pkg/reconciler/decorator/decorator.go

```diff
@@ -219,8 +220,8 @@ func (r *Reconciler) createOrUpdateDecorator(ctx context.Context, decorator *v1a
 	return nil
 }
 
-func (c *Reconciler) resolveDestination(ctx context.Context, destination pkgv1alpha1.Destination, decorator *v1alpha1.Decorator) (string, error) {
-	dest := pkgv1alpha1.Destination{
+func (c *Reconciler) resolveDestination(ctx context.Context, destination duckv1.Destination, decorator *v1alpha1.Decorator) (string, error) {
+	dest := duckv1beta1.Destination{
```

Comment:
Why does this use duckv1beta1.Destination instead of duckv1? Is it because uriResolver still expects duckv1beta1? Seems like that should be fixed in pkg.

Reply:
Yes, that's right.
There should be some follow up change in pkg to receive the v1 duck.